### PR TITLE
Open vino head pose

### PR DESCRIPTION
**Pull request by vinicius121**
This example is estimates the head angles by using Intel OpenVino Inference Engine and the head pose model from open_model_zoo.

### CODE REVIEW

File: SamplesCS/Samples/OpenVinoFaceDetection.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    /// <summary>
+    /// To run this example first you nedd to compile OPENCV with Intel OpenVino
+	/// Download the face detection model available here: https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/face-detection-adas-0001
+    /// Add the files to the bin folder
+    /// </summary>
+    internal class OpenVinoFaceDetection : ISample
+    {
+        public void Run()
+        {
+			const string modelFace = "face-detection-adas-0001.bin";
```
Comment (shimat) [inline SamplesCS/Samples/OpenVinoFaceDetection.cs]: please replace tab with four spaces

**Review by shimat — COMMENTED** [review]
Thank you for your contribution!

File: SamplesCS/Samples/OpenVinoFaceDetection.cs (hunk 1 above)
Comment (vinicius121) [reply SamplesCS/Samples/OpenVinoFaceDetection.cs]: Hi, I think i updated it.

**Comment by shimat** [comment]
Thank you!

**Pull request merged**